src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const nav = document.querySelector('.gallery__list');
+const imgLarge = document.querySelector('.gallery__large-img');
+
+nav.addEventListener('click', (events) => {
+  events.preventDefault();
+
+  const link = events.target.closest('.gallery__thumb');
```

Comment:
The `closest` method should be used to find the closest ancestor that matches the selector. Ensure that the selector is correct and that `link` is indeed the element you want to interact with.